Sources/App/configure.swift

```diff
@@ -32,17 +35,31 @@ public func configure(_ config: inout Config, _ env: inout Environment, _ servic
     middlewares.use(ErrorMiddleware.self) // Catches errors and converts to HTTP response
     services.register(middlewares)
 
-    // Configure a SQLite database
-    let sqlite = try SQLiteDatabase(storage: .memory)
+    let postgresql = try PostgreSQLDatabase(config: dbConfig(environment: Environment.detect()))
 
-    // Register the configured SQLite database to the database config.
+    /// Register the configured PostgreSQL database to the database config.
     var databases = DatabasesConfig()
-    databases.add(database: sqlite, as: .sqlite)
+    databases.add(database: postgresql, as: .psql)
     services.register(databases)
 
     // Configure migrations
     var migrations = MigrationConfig()
-    migrations.prepareCache(for: .sqlite)
-    migrations.add(model: PerformanceTestResults.self, database: .sqlite)
+    migrations.prepareCache(for: .psql)
+    migrations.add(model: PerformanceTestResults.self, database: .psql)
     services.register(migrations)
 }
+
+fileprivate func dbConfig(environment: Environment) throws -> PostgreSQLDatabaseConfig {
```

Comment:
Adapted from vapor cloud v2 docs

Reply:
Actually, could you make this more like the Vapor-TIL book setup? https://github.com/raywenderlich/vapor-til/blob/master/Sources/App/configure.swift#L58 leaves some more flexibility for running locally (I prefer not to use my postgres user for things for example)

Reply:
Updated. Let me know what you think. There were some parts of the vapor-til config func that I didn't like so I opted to update them.